assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -42,36 +48,56 @@ inputs:
       "This is the first part".
     optional: True
+  find_first:
+    type: string
+    description: >-
+      A list of strings to search for in the inference results. The first occurrence of each string will be extracted. 
+      Must provide a comma-separated list of strings.
+      Example:
+      >>> find_first = "positive,negative"
+      >>> completion = "This is a positive example, not negative"
+      # Output: "positive"
+    optional: True
+  extract_number:
+    type: string
+    description: >
+      If the inference results contain a number, this can be used to extract the first or last number in the inference results.
+      The number will be extracted as a string.
+      Example:
+      >>> extract_number = "first"
+      >>> prediction = "Adding 0.3 to 1,000 gives 1,000.3"
+      # Output: "0.3"
+      Example:
+      >>> extract_number = "last"
+      >>> prediction = "Adding 0.3 to 1,000 gives 1,000.3"
+      # Output: "1000.3"
+    optional: True
   regex_expr:
     type: string
     description: >-
       A regular expression to extract the answer from the inference results. The pattern
       must contain a group to be extracted. The first group and the first match will be used. 
       Example: "\n\nThe answer is: (\d)."
     optional: True
-  extract_value_at_index:
-    type: integer
-    description: >-
-      If the regex_expr finds multiple strings matching the pattern in `regex_expr`, this can be used 
-      to extract the preferred value at a given index out of all matched patterns to be used as prediction 
-      value. If omitted, the default behaviour is first matched. 
-    optional: True
-  strip_prefix:
+  strip_characters:
     type: string
     description: >-
-      Characters to remove from the beginning of the extracted answer.
-      It is applied in the very end of the extraction process.
+      A set of characters to remove from the beginning or end of the extracted answer.It is applied in the very end
+      of the extraction process.
     optional: True
-  strip_suffix:
+  label_map:
     type: string
-    description: >-
-      Characters to remove from the end of the extracted answer.
-      It is applied in the very end of the extraction process.
+    description: |
+      JSON serialized dictionary to perform mapping. Must contain key-value pair "column_name": "<actual_column_name>" whose 
+      value needs mapping, followed by key-value pairs containing idtolabel or labeltoid mappers.
+      Example format:
+      {"column_name":"label", "0":"NEUTRAL", "1":"ENTAILMENT", "2":"CONTRADICTION"}
     optional: True
   template:
     type: string
     description: >-
-      Jinja template containing logic to extract prediction.
+      Jinja template containing logic to extract prediction. In case of multiple predictions, logic must be written in a written in
+      format so that it outputs a list of formatted predictions.
```

Comment:
Can we provide a very basic example for it in the description?

Reply:
Done. Thanks.